src/Starlight/Draw/Starfield.hs

```diff
@@ -45,11 +46,13 @@ drawStarfield
      )
   => m ()
 drawStarfield = UI.using getDrawable $ do
-  View{ scale, size, zoom, focus } <- ask
+  View{ size, zoom, focus } <- ask
 
-  resolution_ ?= (fromIntegral <$> size ^* scale)
-  origin_     ?= (prj <$> (focus / (fromIntegral <$> size)))
-  zoom_       ?= zoom
+  resolution_ ?= (fromIntegral <$> size)
+  let V2 (q1, r1) (q2, r2) = (`Fixed.divMod'` 1) . convert <$> focus
+  focusR_     ?= (fmap realToFrac <$> V2 r1 r2)
+  focusQ_     ?= (realToFrac <$> V2 q1 (q2 :: Integer))
+  zoom_       ?= realToFrac zoom
```

Comment:
This one took me a while.

I’m splitting the `focus` into its integral & fractional parts and writing them separately…